var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -13,7 +13,7 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='main')
 
-    version('4.4.36', commit='07fe9999a137c3741fd95713149a76a202cb7d7a')
+    version('4.5.30', commit='3fdb48cad0339cb2eb36d1ad4a7004983525af6a')
```

Comment:
The last version available on devpi is 4.4.59 (https://bbpteam.epfl.ch/repository/devpi/bbprelman/release/bglibpy), I wonder if it would be better to keep them aligned?